deepchem/utils/graph_conv_utils.py

```diff
@@ -0,0 +1,512 @@
+"""
```

Comment:
This file doesn't really have anything to do with convolutions.  How about calling it `molecule_feature_utils.py`?

Reply:
I fixed!